AssetManagementTestProject/TestSetup/Constant.cs

```diff
@@ -4,6 +4,7 @@ public class Constant
 {
     public static string BASE_URL = "https://group4b6ms.azurewebsites.net/";
     public static string BASE_API_URL = "https://group4b6msapi.azurewebsites.net/";
+    public static string LOGIN_PAGE_URl = "https://group4b6ms.azurewebsites.net/login";
```

Comment:
IMO this would be hard to keep track. You can create a private property in the according page obj e.g., In Login Page you can create a `private readonly string loginPath = '/login` and make `Constant.BASE_URL + loginPath`